gradescopeCronJob/gradescope_to_spreadsheet.py

```diff
@@ -98,10 +101,11 @@ def get_sub_sheet_titles_to_ids(sheet_api_instance):
     global subsheet_titles_to_ids
     if subsheet_titles_to_ids:
         return subsheet_titles_to_ids
+    print("Retrieving subsheet titles to ids")
     sheets = sheet_api_instance.get(spreadsheetId=SPREADSHEET_ID, fields='sheets/properties').execute()
-    sub_sheet_titles_to_ids = {sheet['properties']['title']: sheet['properties']['sheetId'] for sheet in
```

Comment:
This PR is to rectify this minor mistake: I accidentally created a new local variable instead of using a similarly named global  variable.

Reply:
Have you tested this?

Reply:
Can you also do me a favor and get rid of `.gitignore` in the root folder, which has `oldScripts/.env` in it. We deleted the oldSripts folder.

Reply:
I have tested it! The script now runs in less than 3 minutes.